rkt/metadata_service.go

```diff
@@ -455,16 +454,51 @@ func logReq(h func(w http.ResponseWriter, r *http.Request)) http.HandlerFunc {
 	}
 }
 
-func makeHandlers() http.Handler {
+// unixListener returns the listener used for registrations (over unix sock)
+func unixListener() (net.Listener, error) {
+	s := os.Getenv("LISTEN_FDS")
+	if s != "" {
+		// socket activated
+		lfds, err := strconv.ParseInt(s, 10, 16)
+		if err != nil {
+			return nil, fmt.Errorf("Error parsing LISTEN_FDS env var: %v", err)
```

Comment:
@jonboulle the errors should start lowercase as they're usually appended to other messages as the stack unwinds. Since this is private to "main", it was my intention to just print them "as is" in main. That's why they're capitalized. Unfortunately I still appended it out of habit :) I'll remove the prefix in "main".